### ISSUE TITLE
DK - Unholy - Lacks Death Strike logic (QoL)

### ISSUE THREAD

**Issue by idubs** [herotc/hero-rotation#260]
Although the addon options support Death Strike in solo mode the unholy rotation lacks any Death Strike logic

**Comment by zaberwookie** [comment]
yes and wake of ashes for ret pallys among others, i made a post about this already, no response https://github.com/herotc/hero-rotation/issues/255

**Comment by idubs** [comment]
It also looks like unholy has no range or target checks, I'm going to see if I can find some time to revamp the existing logic and make a pull request.

**Comment by chrislopez24** [comment]
Hi! DK maintainer here!
Thanks for the suggestion! DS for solo mode was added by a contributor, not me. It will be added in the future when i find time, but its not a priority for me atm. I will keep this open for reference

**Comment by FluffyDK** [comment]
i will add it also to unholy
